test/test_framework.py

```diff
@@ -691,93 +653,41 @@ class SomeObject(Object):
-        ), (
-            {"1"},
-            {"1", "2"},
-            lambda a, b: a > b,
-            False,
-            True
-        ), (
-            # Empty set comparison.
-            set(),
-            set(),
-            lambda a, b: a == b,
-            True,
-            True
-        ), (
-            {"a", "c"},
-            {"c", "a"},
-            lambda a, b: a == b,
-            True,
-            True
-        ), (
-            dict(),
-            dict(),
-            lambda a, b: a == b,
-            True,
-            True
-        ), (
-            {"1": "2"},
-            {"1": "2"},
-            lambda a, b: a == b,
-            True,
-            True
-        ), (
-            {"1": "2"},
-            {"1": "3"},
-            lambda a, b: a == b,
-            False,
-            False
-        ), (
-            [],
-            [],
-            lambda a, b: a == b,
-            True,
-            True
-        ), (
-            [1, 2],
-            [1, 2],
-            lambda a, b: a == b,
-            True,
-            True
-        ), (
-            [1, 2, 5, 6],
-            [1, 2, 5, 8, 10],
-            lambda a, b: a <= b,
-            True,
-            False
-        ), (
-            [1, 2, 5, 6],
-            [1, 2, 5, 8, 10],
-            lambda a, b: a < b,
-            True,
-            False
-        ), (
-            [1, 2, 5, 8],
-            [1, 2, 5, 6, 10],
-            lambda a, b: a > b,
-            True,
-            False
-        ), (
-            [1, 2, 5, 8],
-            [1, 2, 5, 6, 10],
-            lambda a, b: a >= b,
-            True,
-            False
-        )]
+        test_operations = [
+            (
+                {"1"},  # Operand A.
+                {"1", "2"},  # Operand B.
+                lambda a, b: a < b,  # Operation to test.
+                True,  # Result of op(A, B).
+                False,  # Result of op(B, A).
+            ),
+            ({"1"}, {"1", "2"}, lambda a, b: a > b, False, True),
+            (
+                # Empty set comparison.
+                set(),
+                set(),
+                lambda a, b: a == b,
+                True,
+                True,
+            ),
+            ({"a", "c"}, {"c", "a"}, lambda a, b: a == b, True, True),
+            (dict(), dict(), lambda a, b: a == b, True, True),
+            ({"1": "2"}, {"1": "2"}, lambda a, b: a == b, True, True),
+            ({"1": "2"}, {"1": "3"}, lambda a, b: a == b, False, False),
+            ([], [], lambda a, b: a == b, True, True),
+            ([1, 2], [1, 2], lambda a, b: a == b, True, True),
+            ([1, 2, 5, 6], [1, 2, 5, 8, 10], lambda a, b: a <= b, True, False),
+            ([1, 2, 5, 6], [1, 2, 5, 8, 10], lambda a, b: a < b, True, False),
+            ([1, 2, 5, 8], [1, 2, 5, 6, 10], lambda a, b: a > b, True, False),
+            ([1, 2, 5, 8], [1, 2, 5, 6, 10], lambda a, b: a >= b, True, False),
```

Comment:
Same.. the test was organized with readability in mind, and it all looks reasonable. It completely ignored the choice of formatting and blind-reformatted it based only on line length.